calico.go

```diff
@@ -178,6 +178,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 			endpoint = api.NewWorkloadEndpoint()
 			endpoint.Metadata.Name = args.IfName
 			endpoint.Metadata.Node = nodename
+			endpoint.Metadata.ActiveInstanceID = args.ContainerID
```

Comment:
we populate that field in `k8s.CmdAddK8s`, we want the same behavior for non-k8s orchestrators, this is for that.